plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -15,13 +15,16 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Throwables;
```

Comment:
optional: Is it possible to add a test to TestDeltaLakeMetadata? The newly added tests run only on cloud. It's a little hard to test the detailed behavior quickly.